### PR TITLE
[bitnami/thanos] Add revisionHistoryLimit option for each component

### PR DESCRIPTION
**Pull request by jakuboskera**
### Description of the change

Adds revisionHistoryLimit option for each Thanos component

### Benefits

Able to select how many revisions I want to keep in Kubernetes cluster for each component, e.g. when using GitOps there is no need to have revisionHistoryLimit greater than 1 as rollback is done using Git.

### Possible drawbacks

none

### Applicable issues

none

### Additional information

none

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/thanos/Chart.yaml [hunk 1]
```diff
@@ -35,4 +35,4 @@ maintainers:
 name: thanos
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/thanos
-version: 12.21.2
+version: 12.21.3
```
Comment (FraPazGal) [inline bitnami/thanos/Chart.yaml:38]: We should bump the minor version per semver policy as we are introducing a new feature.

```suggestion
version: 12.22.0
```

**Review by FraPazGal — COMMENTED** [review]
Thanks for the contribution @jakuboskera! I added a minor comment as a review, please check it when you have some time.

File: bitnami/thanos/Chart.yaml (hunk 1 above)
Comment (jakuboskera) [reply bitnami/thanos/Chart.yaml:38]: Yeah that's true, I was about that it is part of almost all Helm Charts, and it is missing here, so I bumped the minor. But I leave the decision up to you.

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**